tests/Feature/Api/CityControllerTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api;
+
+use App\Models\City;
+use App\Models\Country;
+use App\Models\User;
+use Laravel\Sanctum\Sanctum;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\TestCase;
+
+class CityControllerTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $adminUser = User::factory()->create();
+        Sanctum::actingAs($adminUser, ["HasAdminRole"]);
```

Comment:
```suggestion
        $this->actingAs($adminUser);
```
should be enough, I think.